### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by spathak98**
@spielers I have updated the file, Please look in to it.

### CODE REVIEW

**Comment by spielers** [comment]
@spathak98 have you aiming for Hacktoberfest ?? Do let us know will add the label to it.

Best
spielers

**Comment by spathak98** [comment]
hey @spielers  I am aiming for Hacktoberfest, so please add Hacktoberfest label.

File: docs/tutorial/part-two/index.md [hunk 1]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
+Gatsby ट्यूटोरियलच्या भाग दोन मध्ये आपले स्वागत आहे!
 
-## What's in this tutorial?
+## या ट्यूटोरियल मध्ये काय आहे?
 
-In this part, you're going to explore options for styling Gatsby websites and dive deeper into using React components for building sites.
+या भागात आपण गॅट्सबी वेबसाइटच्या स्टाईलिंगसाठी पर्याय शोधून काढत आहात आणि साइट तयार करण्यासाठी घटकांच्या प्रतिक्रियांचा वापर करुन आणखी खोलवर जा.
 
-## Using global styles
+## जागतिक शैली वापरणे
```
Comment (spielers) [inline docs/tutorial/part-two/index.md]: शैली वापरणे = >ग्लोबल स्टईल्स

File: docs/tutorial/part-two/index.md [hunk 2]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
 
-Every site has some sort of global style. This includes things like the site's typography and background colors. These styles set the overall feel of the site — much like the color and texture of a wall sets the overall feel of a room.
+प्रत्येक साइटवर काही प्रमाणात जागतिक शैली असते. यात साइटचे टायपोग्राफी आणि पार्श्वभूमी रंग यासारख्या गोष्टी समाविष्ट आहेत. या शैलींनी साइटची संपूर्ण भावना निश्चित केली आहे - भिंतीचा रंग आणि पोत जसे खोलीचा एकंदर अनुभव सेट करते.
```
Comment (spielers) [inline docs/tutorial/part-two/index.md]: ग्लोबल स्टईल्स use this

File: docs/tutorial/part-two/index.md [hunk 3]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
 
-### Creating global styles with standard CSS files
+### मानक सीएसएस फायलीसह जागतिक शैली तयार करीत आहे
```
Comment (spielers) [inline docs/tutorial/part-two/index.md]: ग्लोबल स्टईल्स

File: docs/tutorial/part-two/index.md [hunk 4]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
 
-One of the most straightforward ways to add global styles to a site is using a global `.css` stylesheet.
+एखाद्या साइटवर जागतिक शैली जोडण्याचा सर्वात सोपा मार्ग म्हणजे एक ग्लोबल वापरणे `.css` स्टाईलशीट.
 
-#### ✋ Create a new Gatsby site
+#### ✋ नवीन गॅटस्बी साइट तयार करा
 
-Start by creating a new Gatsby site. It may be best (especially if you're new to the command line) to close the terminal windows you used for [part one](/tutorial/part-one/) and start a new terminal session for part two.
+नवीन तयार करून प्रारंभ करा Gatsby site. हे सर्वोत्तम असू शकते (विशेषत: जर आपण कमांड लाइनमध्ये नवीन असाल) आपण वापरलेल्या टर्मिनल विंडो बंद करण्यासाठी [part one](/tutorial/part-one/) आणि भाग दोन साठी नवीन टर्मिनल सत्र प्रारंभ करा.
```
Comment (spielers) [inline docs/tutorial/part-two/index.md]: नवीन तयार करून प्रारंभ करा Gatsby site.  === > please retanslate this

File: docs/tutorial/part-two/index.md [hunk 5]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
 
-Open a new terminal window, create a new "hello world" Gatsby site in a directory called `tutorial-part-two`, and then move to this new directory:
+एक नवीन टर्मिनल विंडो उघडा, `tutorial-part-two` नावाच्या निर्देशिकेत एक नवीन "hello world" Gatsby site तयार करा आणि नंतर या नवीन निर्देशिकेत जा:
```
Comment (spielers) [inline docs/tutorial/part-two/index.md]: directory  == > डिरेक्टरी  it should be the same as it is.

File: docs/tutorial/part-two/index.md [hunk 6]
```diff
@@ -48,9 +48,9 @@ You now have a new Gatsby site (based on the Gatsby "hello world" starter) with
 │       └── index.js
 ```
 
-#### ✋ Add styles to a CSS file
+#### ✋ CSS फाइलमध्ये शैली जोडा
```
Comment (spielers) [inline docs/tutorial/part-two/index.md]: शैली ==> स्टईल्स

File: docs/tutorial/part-two/index.md [hunk 7]
```diff
@@ -59,9 +59,9 @@ cd styles
 touch global.css
 ```
 
-> Note: Feel free to create these directories and files using your code editor, if you'd prefer.
+> टीप: आपण इच्छित असल्यास आपल्या कोड संपादकाचा वापर करून या निर्देशिका आणि फायली तयार करा.
```
Comment (spielers) [inline docs/tutorial/part-two/index.md]: निर्देशिका ==> should be a directory

File: docs/tutorial/part-two/index.md [hunk 8]
```diff
@@ -296,36 +296,36 @@ export default () => (
...
 
-The finished page should now look like:
+तयार झालेले पृष्ठ आता यासारखे दिसले पाहिजे:
 
 ![User list page with CSS modules](css-modules-userlist.png)
 
 ### CSS-in-JS
 
-CSS-in-JS is a component-oriented styling approach. Most generally, it is a pattern where [CSS is composed inline using JavaScript](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js).
+CSS-in-JS हा घटक-आधारित स्टाईल दृष्टीकोन आहे. सामान्यत :, हा एक नमुना आहे जेथे जावास्क्रिप्ट वापरुन [CSS is composed inline using JavaScript](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js).
```
Comment (spielers) [inline docs/tutorial/part-two/index.md]: जावास्क्रिप्ट == > it should be in english

**Review by spielers — CHANGES_REQUESTED** [review]
Hey, @spathak98 please do the required changes

Best,
spielers

File: docs/tutorial/part-two/index.md [hunk 9]
```diff
@@ -114,43 +114,43 @@ import "./src/styles/global.css"
...
+> डीफॉल्टनुसार स्थानिक पातळीवर स्कोप केलेले आहेत.
 
-CSS Modules are very popular because they let you write CSS normally but with a lot more safety. The tool automatically generates unique class and animation names, so you don't have to worry about selector name collisions.
+CSS मॉड्यूल खूप लोकप्रिय आहेत कारण ते आपल्याला सामान्यपणे CSS लिहू देतात परंतु बर्‍याच सुरक्षिततेसह. साधन स्वयंचलितपणे अद्वितीय वर्ग आणि अ‍ॅनिमेशन नावे व्युत्पन्न करते, म्हणून आपल्याला निवडकर्त्याच्या नावाच्या टक्करांची चिंता करण्याची आवश्यकता नाही.
 
-Gatsby works out of the box with CSS Modules. This approach is highly recommended for those new to building with Gatsby (and React in general).
+Gatsby CSS मॉड्यूलसह बॉक्सच्या बाहेर कार्य करते. Gatsby (आणि सर्वसाधारणपणे प्रतिक्रिया द्या) सह नवीन बांधकाम करणार्‍यांसाठी हा दृष्टिकोन अत्यंत शिफारसीय आहे.
 
-#### ✋ Build a new page using CSS Modules
+#### ✋ CSS मॉड्यूल्स वापरुन एक नवीन पृष्ठ तयार करा
```
Comment (visshaljagtap) [inline docs/tutorial/part-two/index.md]: dont translate page, keep as is

**Review by visshaljagtap — CHANGES_REQUESTED** [review]

**Comment by spathak98** [comment]
Hey @spielers I have changed the required changes, please look into it.

**Comment by spielers** [comment]
Hey, @visshaljagtap accepted for merging?

**Comment by visshaljagtap** [comment]
Yes @spielers 
Thanks for the contribution @spathak98 

LGTM!

**Review by visshaljagtap — APPROVED** [review]

**Review by spielers — APPROVED** [review]
Cheers 😊!

**Pull request merged**